api_commits.go

```diff
@@ -69,6 +69,7 @@ func (a *CommitsApiService) CheckoutCommit(ctx _context.Context, repositoryName
 	if localVarHTTPHeaderAccept != "" {
 		localVarHeaderParams["Accept"] = localVarHTTPHeaderAccept
 	}
+	var err error
```

Comment:
Is this required? usually the following `r, err :=` will handle creating an error object for err

Reply:
The issue is the line in cases where you do have headers:

```
	var err error
	localVarHeaderParams["titan-remote-parameters"], err = parameterToJson(titanRemoteParameters)
```

You can't both assign a member of a map and declare a variable with ":=". Declaring a new temporary variable would require giving it a unique name (in case you have mutiple headers). It's harmless to pre-declare it, and seemed like the easiest way.